<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TricksDialog</class>
 <widget class="QDialog" name="TricksDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1005</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Enable Tricks</string>
  </property>
  <widget class="QDialogButtonBox" name="bbox_tricks">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>270</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>10</y>
     <width>991</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="vlay_tricks">
    <item>
     <layout class="QHBoxLayout" name="hlay_tricks_body">
      <item>
       <widget class="QListView" name="disabled_tricks">
        <property name="toolTip">
         <string>test</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="vlay_tricks_controls">
        <item>
         <widget class="QPushButton" name="disable_trick">
          <property name="text">
           <string>Disable
&lt;--</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="enable_trick">
          <property name="text">
           <string>Enable
--&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListView" name="enabled_tricks"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>bbox_tricks</sender>
   <signal>accepted()</signal>
   <receiver>TricksDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bbox_tricks</sender>
   <signal>rejected()</signal>
   <receiver>TricksDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
